docs/sql/json.md

```diff
@@ -7,28 +7,20 @@ Any such value holds at runtime two pieces of information:
 - the data type
 - the data value
 
-::: warning
-
-Currently `VARIANT` values are not supported in table or output views;
-they can only be used in intermediate computations.
-
-:::
-
-
 Values of `VARIANT` type can be created by casting any other value to
-a `VARIANT`: e.g.  `SELECT CAST(x AS VARIANT)`.  Conversely, values of
-type `VARIANT` can be cast to any other data type `SELECT CAST(variant
-AS INT)`.  A cast of a value of type `VARIANT` to target type T will
-compare the runtime type with T.  If the types are identical or there
-is a natural conversion from the runtime type to T, the original value
+a `VARIANT`: e.g. `SELECT CAST(x AS VARIANT)`.  Conversely, values of
+type `VARIANT` can be cast to any other data type`SELECT CAST(variant AS INT)`.
+A cast of a value of type `VARIANT` to target type `T` will
+compare the runtime type with `T`.  If the types are identical or there
```

Comment:
I don't think we convert to strings yet.
could be done, though

Reply:
Not sure what you mean. Which conversion is missing?

Reply:
Anything to string

Reply:
Even if the variant contains string? What can you cast it to then?

Reply:
If it does not contain string